kinto/core/resource/__init__.py

```diff
@@ -599,7 +613,7 @@ def process_record(self, new, old=None):
 
         return new
 
-    def apply_changes(self, record, changes):
+    def apply_changes(self, record, changes, operations=None):
```

Comment:
> I think we should just "bite the bullet" and make a breaking change. Have apply_changes take the "operations" (I think that's a better name) and have it return (updated, changes).

We should change the return value only when we have a JSON-Patch, right? Or should we do it also for regular calls? Just to make sure that I've got it right, the approach would be:

- Keep the signature ``def apply_changes(self, record, changes)``
- In case of a regular call, ``return updated``
- If Json Patch, ``return (updated, changes)``

Is that clear?